quinn-h3/benches/bench.rs

```diff
@@ -109,29 +118,38 @@ impl Context {
         Self {
             server_config: server::Builder::with_quic_config(server_config),
             client_config: client::Builder::with_quic_config(client_config),
+            stop_server: None,
         }
     }
 
-    pub fn spawn_server(&self) -> (SocketAddr, thread::JoinHandle<()>) {
+    pub fn spawn_server(&mut self) -> (SocketAddr, thread::JoinHandle<()>) {
         // TODO: Let the OS choose a free port for us
```

Comment:
@Ralith I'm not sure how this is possible. In the `quinn` bench, you use the port `0`, but how does the client know to which port it should connect ?

Reply:
The trick is to bind a socket with port 0, then ask the OS what port it chose, then pass that to the client. I think this was covered in gitter earlier this morning.